### DISCUSSION TITLE
📚 Documentation Improvement for Tamagui React Native UI Kit 🚀

### DISCUSSION THREAD

**Discussion by khubaib-11** [tamagui/tamagui#1889] [General]
Issue Description:
Hey Tamagui Team! 👋

I hope this message finds you well. First off, let me express my appreciation for the incredible work you've put into Tamagui – it's a fantastic React Native UI Kit. 🙌

However, after spending some time exploring the documentation, I noticed a few areas that could use improvement. The docs, while comprehensive, tend to assume a certain level of pre-existing knowledge. This might be challenging for users who are new to Tamagui or React Native in general. 🤔

Here are a couple of specific concerns:

**1. Assumed Prior Knowledge:**
Some sections assume that the reader is already familiar with certain concepts. It would be immensely helpful to include more introductory content or links to relevant resources for users who may not be experts in React Native. 📖

**2. Clarity in Examples:**
While the documentation provides examples, there are instances where a bit more explanation or comments within the code could enhance clarity. This would greatly benefit users who are trying to implement features for the first time. 💡

I believe addressing these concerns would significantly enhance the onboarding experience for new users and contribute to the overall usability of Tamagui. I'd be happy to collaborate or provide more specific examples if needed. 🤝

Thank you for your attention to this matter, and keep up the excellent work! 🚀

Best regards, Khubaib

**Comment by natew** [comment]
Yep, the docs could use many iterations. We have a PR open with a re-structure, but we may end up redoing some of it so still needs a bit of time. I appreciate the kind words! Moving to discussion.

**Comment by khubaib-11** [comment]
A simple insight that making docs language easy can greatly boosts good user experience 🫰:

=== 📚 Actual Tamagui Docs ===

**Installation:**
We recommend using the CLI  to bootstrap one of the example apps, whether you use it directly or just as a reference when implementing in your stack.


npm create tamagui@latest


==== 🌟 Simplified Language ====

We suggest using the CLI 🚀 to start with one of the sample apps. You can use the sample app as it is or change it to fit your needs.

To use the CLI, type this in your terminal:


npm create tamagui@latest
